src/server/cmd/pachd/main.go

```diff
@@ -184,14 +173,19 @@ func doEnterpriseMode(config interface{}) (retErr error) {
 	} else {
 		log.Printf("no Jaeger collector found (JAEGER_COLLECTOR_SERVICE_HOST not set)")
 	}
-	env := serviceenv.InitWithKube(serviceenv.NewConfiguration(config))
-	profileutil.StartCloudProfiler("pachyderm-pachd-enterprise", env.Config())
+	env = serviceenv.InitWithKube(serviceenv.NewConfiguration(config))
+	profileutil.StartCloudProfiler(service, env.Config())
```

Comment:
I wonder if we should move this below the `log.SetFormatter` call, since it logs.  (I also have a feeling that `automaxprocs` logs pretty early as well... maybe we need to do the same there.)